onnx_diagnostic/torch_models/hghub/hub_api.py

```diff
@@ -10,6 +10,7 @@
 from ...helpers.config_helper import update_config
 from . import hub_data_cached_configs
 from .hub_data import __date__, __data_tasks__, load_architecture_task, __data_arch_values__
+import diffusers
```

Comment:
Can you import it when you need it? importing this package takes a while and I prefer to wait only when it is needed.